sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs

```diff
@@ -671,7 +671,7 @@ public partial class ContinuousWebJobData : Azure.ResourceManager.AppService.Mod
         public string ExtraInfoUrl { get { throw null; } set { } }
         public string LogUrl { get { throw null; } set { } }
         public string RunCommand { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, object> Settings { get { throw null; } }
+        public System.Collections.Generic.IDictionary<string, System.BinaryData> Settings { get { throw null; } }
```

Comment:
Similar as one of the comment above, it is hard to believe that this `Settings` is a dictionary of a dictionary.
We might need to change this back to `Dictionary<string, object>`